frontend/src/components/AdminControls/Recipes.js

```diff
@@ -111,6 +111,8 @@ const Recipes = ({ recipes }) => {
 
   const options = {
     hideSizePerPage: true,
+    alwaysShowAllBtns: true,
+    withFirstAndLast: false,
```

Comment:
Discussed this in person, but think these settings also need to be applied to the user table, as well.